client/my-sites/customer-home/welcome-banner.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import DismissibleCard from 'blocks/dismissible-card';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CardHeading from 'components/card-heading';
+import { bumpStat, composeAnalytics, recordTracksEvent } from 'state/analytics/actions';
+
+export const WelcomeBanner = ( { translate, recordStats } ) => {
+	return (
+		<DismissibleCard
```

Comment:
Should we also check for user age? I think any user created before ` new Date( '2019-08-06' )` should see this, but any users created after this date already have Customer Home.

Most folks are single site users, so I don't think we need to worry about per-site gating.

Reply:
> Should we also check for user age?

I think so. Reading through #39178 seems this banner was intended for old sites only. Sites that already see this page shouldn't see the banner:

> With Customer Home being enabled for all Simple and Atomic sites in #39071, older sites need something to introduce them to this new page they haven't seen before

> We can use a full width banner across the top that shows only for sites created before August 2019

Reply:
Yeah, let's add a user age check. The welcome banner isn't necessary for folks who already have Customer Home active.

Reply:
5da75501575493b1609418714c294d6771f6e2bb